recipe/conda_build_config.yaml

```diff
@@ -6,7 +6,10 @@ docker_image:                             # [linux64]
   - condaforge/linux-anvil-cos7-x86_64    # [linux64]
 cdt_name:                                 # [linux64]
   - cos7                                  # [linux64]
-MACOSX_DEPLOYMENT_TARGET:  # [osx and x86_64]
+c_stdlib_version:          # [osx and x86_64]
   - "10.13"                # [osx and x86_64]
 MACOSX_SDK_VERSION:        # [osx and x86_64]
   - "10.13"                # [osx and x86_64]
+
+c_stdlib_version:   # [linux]
+  - "2.17"          # [linux]
```

Comment:
@h-vetinari, @beckermr, this is a minor thing, but just wanted to make you aware in case this hints at other potential issues:
The selector here is `[linux]` whereas it was originally `[linux64]` for the `sysroot_linux-64 2.17` requirement.

Since we use glibc 2.17 for `linux-aarch64`/`linux-ppc64le` anyway, this is fine of course.
(But it could also be an indicator on some missed corner cases for the migrator, hence pinging you on this.)

Reply:
This part is correct. While the sysroot requirement for 2.17 in meta.yaml was only for linux-64, aarch/ppc are _already_ on 2.17, so it makes sense to write a linux-wide requirement in the CBC.